brayns/engine/Engine.h

```diff
@@ -38,20 +40,18 @@ class Engine
 {
 public:
     /**
-     * @brief Initializes OSPRay and register core engine objects available for use (cameras, renderers, materials
+     * @brief Initializes Ospray and register core engine objects available for use (cameras, renderers, materials
```

Comment:
Replace all artifact or is it the correct spelling ?

Reply:
I meant to call it as Ospray from now on. I fixed all occourences, thanks.